operation/fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/model/type/JobParameter.java

```diff
@@ -475,11 +500,11 @@ public static void parse(Builder builder, JsonObject obj) throws FHIROperationEx
 
     @Override
     public String toString() {
-        return "JobParameter [fhirResourceType=" + fhirResourceType + ", fhirSearchFromDate=" + fhirSearchFromDate
-                + ", fhirTenant=" + fhirTenant + ", fhirDataStoreId=" + fhirDataStoreId + ", fhirPatientGroupId="
-                + fhirPatientGroupId + ", fhirTypeFilters=" + fhirTypeFilters + ", cosBucketName=" + cosBucketName
-                + ", cosLocation=" + cosLocation + ", cosEndpointUrl=" + cosEndpointUrl + ", cosCredentialIbm="
-                + cosCredentialIbm + ", cosApiKey=" + cosApiKey + ", cosSrvInstId=" + cosSrvInstId
-                + ", cosBucketPathPrefix=" + cosBucketPathPrefix + "]";
+        return "JobParameter [fhirResourceType=" + fhirResourceType + ", fhirSearchFromDate=" + fhirSearchFromDate + ", fhirTenant=" + fhirTenant
+                + ", fhirDataStoreId=" + fhirDataStoreId + ", fhirPatientGroupId=" + fhirPatientGroupId + ", fhirTypeFilters=" + fhirTypeFilters
+                + ", fhirExportFormat=" + fhirExportFormat + ", inputs=" + inputs + ", storageDetail=" + storageDetail + ", incomingUrl=" + incomingUrl
+                + ", cosBucketName=" + cosBucketName + ", cosOperationBucketNameOo=" + cosOperationBucketNameOo + ", cosLocation=" + cosLocation
+                + ", cosEndpointUrl=" + cosEndpointUrl + ", cosCredentialIbm=" + cosCredentialIbm + ", cosApiKey=" + cosApiKey + ", cosSrvInstId="
+                + cosSrvInstId + ", cosBucketPathPrefix=" + cosBucketPathPrefix + "]";
```

Comment:
minor formatting suggestion, take or leave:
```suggestion
        return "JobParameter [fhirResourceType=" + fhirResourceType + 
                ", fhirSearchFromDate=" + fhirSearchFromDate + 
                ", fhirTenant=" + fhirTenant + 
                ", fhirDataStoreId=" + fhirDataStoreId + 
                ", fhirPatientGroupId=" + fhirPatientGroupId + 
                ", fhirTypeFilters=" + fhirTypeFilters + 
                ", fhirExportFormat=" + fhirExportFormat + 
                ", inputs=" + inputs + 
                ", storageDetail=" + storageDetail + 
                ", incomingUrl=" + incomingUrl + 
                ", cosBucketName=" + cosBucketName + 
                ", cosOperationBucketNameOo=" + cosOperationBucketNameOo + 
                ", cosLocation=" + cosLocation + 
                ", cosEndpointUrl=" + cosEndpointUrl + 
                ", cosCredentialIbm=" + cosCredentialIbm + 
                ", cosApiKey=" + cosApiKey + 
                ", cosSrvInstId=" + cosSrvInstId + 
                ", cosBucketPathPrefix=" + cosBucketPathPrefix + "]";
```